scripts/runClientBuild.js

```diff
@@ -1,16 +1,12 @@
+import 'colors';
 import async from 'async';
-import colors from 'colors';
-import path from 'path';
 import Table from 'cli-table';
 import webpack from 'webpack';
-import lodash from 'webpack';
 
 import webpackClientFullConfig from '../webpack/clientFull';
 import webpackClientFullMinConfig from '../webpack/clientFullMin';
 import webpackClientNoReactConfig from '../webpack/clientNoReact';
 import webpackClientNoReactMinConfig from '../webpack/clientNoReactMin';
```

Comment:
I'm okay keeping them for now, as at least No-React one is used in Apiary and others in tests